virt/vm_networking/virt-connecting-vm-to-default-pod-network.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="virt-connecting-vm-to-default-pod-network"]
+= Connecting a virtual machine to the default pod network
+include::_attributes/common-attributes.adoc[]
+:context: virt-connecting-vm-to-default-pod-network
+
+toc::[]
+
+You can connect a virtual machine to the default internal pod network by configuring its network interface to use the `masquerade` binding mode.
+
+[NOTE]
+====
+Traffic passing through network interfaces to the default pod network is interrupted during live migration.
+====
+
+include::modules/virt-configuring-masquerade-mode-cli.adoc[leveloffset=+1]
+
+include::modules/virt-configuring-masquerade-mode-dual-stack.adoc[leveloffset=+1]
+
+// TO DO: This should be moved to an optimization section
+include::modules/virt-jumbo-frames-vm-pod-nw.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_virt-connecting-vm-to-default-pod-network"]
+.Additional resources
```

Comment:
Given that this is an assembly-level  Additional resources section, I believe this should be heading and not a label.